tabular/src/autogluon/tabular/models/rf/rf_model.py

```diff
@@ -27,12 +25,25 @@ class RFModel(AbstractModel):
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
         self._feature_generator = None
+        self._daal = False  # Whether daal4py backend is being used
 
     def _get_model_type(self):
+        if self.problem_type == QUANTILE:
+            from .rf_quantile import RandomForestQuantileRegressor
+            return RandomForestQuantileRegressor
+        if self.params_aux.get('use_daal', False):
+            # Disabled by default because it appears to degrade performance
+            try:
+                from daal4py.sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
+                self._daal = True
```

Comment:
Missing analogous statement you had for KNN:

`logger.log(15, '\tUsing daal4py RF backend...')`

Reply:
Added